package.json

```diff
@@ -57,7 +57,6 @@
     "reflect-metadata": "0.1.13",
     "rimraf": "3.0.2",
     "rxjs": "7.0.0",
-    "shortid": "2.2.16",
```

Comment:
Good thing that we dropped this.

> shortid is deprecated, because the architecture is unsafe. we instead recommend Nano ID, which has the advantage of also being significantly faster than shortid